query-node/mappings/src/membership.ts

```diff
@@ -655,6 +656,18 @@ async function processMemberRemark(
 ): Promise<Partial<MetaprotocolTransactionSuccessful>> {
   const messageType = decodedMessage.memberRemarked
 
+  if (messageType === 'createApp') {
+    await processCreateAppMessage(store, event, decodedMessage.createApp!, memberId.toString())
+  }
+
+  if (messageType === 'updateApp') {
+    await processUpdateAppMessage(store, decodedMessage.updateApp!)
+  }
+
+  if (messageType === 'deleteApp') {
+    await processDeleteAppMessage(store, decodedMessage.deleteApp!)
+  }
+
```

Comment:
In all those if-blocks there is no `return` statement, so the function continues until it hits:
```typescript
  // unknown message type
  return inconsistentState('Unsupported message type in member_remark action', messageType)
```